.github/pull_request_template.md

```
@@ -44,6 +47,10 @@ Fixes #<!--Add related issue number here and delete this comment text block.-->
 
 <!-- If this PR introduces a breaking change, please describe the impact and a migration path for existing applications. -->
 
+**Checklist for Repository Standards**
```

Markdownlint (0.35.0)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)